### ISSUE TITLE
[Accessibility]: "Finish this application" button styled like a link

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#6061]
### Point of contact

jeana-adhoc

### How severe do you think this issue is?

Medium

### Severity rationale

I marked this as a medium because it has been in existence for years. 

### VA 508 Office

N/A

### URL where issue occurs

https://staging.va.gov/records/request-personal-records-form-20-10206/preparer-type

### Component or pattern (if applicable)

This is within the forms library and is part of the save in progress feature.

however it is not a component, it is hard coded button, that looks like a link. This is also then considered an imposter component.

### Environment details

This happens in any browser and any screen reader

### Steps to reproduce

1. Navigate to https://staging.va.gov/records/request-personal-records-form-20-10206/preparer-type
2. Inspect the "Finish this request later" link

And see this code

```html
<div class="vads-u-display--flex vads-u-margin-top--2">
   <button type="button" class="va-button-link schemaform-sip-save-link">
      Finish this request later
   </button>
</div>
```

### Expected behavior / Acceptance criteria

Since it looks like a link, I would expect the underlying code to support that. 

The role (button) doesn't match the perceived purpose, of navigating.

However, it is doing 2 things.
- It is saving the form in progress
- AND Navigating to a `form-saved` page.

### Related issues or systemic notes

[This ticket started to capture some of the issues](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1905) (But is now frozen in a migrated repo) 

In December 2024 I wrote

The Accessibility CoP and Design system, were recently examining the "Finish this application later" text that appears above the back/continue buttons.

[DSVA Thread](https://dsva.slack.com/archives/C01DBGX4P45/p1733433690468309)

@humancompanion-usds said

> va-button-link is an oxymoron and thus a bug. Material honesty, especially in this case. If that link should be a button then it should appear as a button and not a link in sheep's clothing. We have tertiary button style that we could use. So there is some design work to be done here. Do you prefer to take that on or should we make that a future forms task?

I agreed to spike on this to at least get some data documented on how we got to this point.

### Other existing tickets
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/25111
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/66873 (This ticket contradicts what we've been finding - and needs to be evaluated)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/103783 - Identified this link as an imposter component
- - https://github.com/department-of-veterans-affairs/vets-website/pull/37622 - @Mottie committed a fix that allows this link to *actually* be a link by adding the following code to their form config, however teams must manually do this
```
formOptions: {
    useWebComponentForNavigation: true,
  },
```
- Meanwhile save-in-progress research began and we changed the design of this link/button to be a button, and put it along side the continue button in the form in the minimal form flow 
<img width="293" height="38" alt="Image" src="https://github.com/user-attachments/assets/3417029b-bb7e-40b9-9856-8393db025741" />
 
  - This tested well. But decisions need to be made whether or not to implement this design or stay with the link.

### Definition of done
- [X] Document timeline of history
   - [x]  This is complete, and documented here: https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Accessibility/save-in-progress.md
- [ ] Figure out if we still agree with where we are today
- [ ] Figure out if any changes are needed

### Type of issue

- [ ] Axe-core
- [x] Cognitive | user flows & navigating
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else

**Comment by jeana-adhoc** [comment]
To design system team - whoever picks this up - let's meet about this first.
